pkg/collector/corechecks/sbom/check.go

```diff
@@ -3,7 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2022-present Datadog, Inc.
 
-//go:build trivy
+//go:build trivy || windows
```

Comment:
Could we add the trivy build flag to windows instead ? I think the `windows` build flag can be used by the cluster-agent on windows